src/__tests__/Voice.test.ts

```diff
@@ -39,4 +39,14 @@ describe("Voice operations", () => {
       throw new Error("test failed");
     }
   });
+
+  test("It should list all of the voices that match the filtering parameters", async () => {
+    try {
+      const voices: any = await Voice.list({ tags: "fun", gender: "male", language: "english" });
+      expect(Array.isArray(voices.voices)).toBe(true);
```

Comment:
hey thanks for the update, it looks far better now! just one more thing, do you think we should at least compare the number of voices filtered/non-filtered?
the other test without filters can save that number to a variable, then at the end of this test with filtering enabled, we compare two and say `expect(voices.voices.length).toBeLessThan(allVoicesCount)`

Reply:
compared

Reply:
I added toBeLessOrEqual, just in case the filtering parameter is the same for all voices. Will never happen I think but it is an edge case covered